packages/client/src/raffles/types/postRaffleParticipation.types.ts

```diff
@@ -0,0 +1,18 @@
+import type { Config } from '../..';
+import type { Raffle, RaffleParticipation } from './raffles.types';
+
+export type PostParticipationData = Pick<
+  RaffleParticipation,
+  | 'userId'
+  | 'billingAddressId'
+  | 'shippingAddressId'
+  | 'productVariantId'
+  | 'paymentTokenId'
+  | 'trackingCorrelationId'
```

Comment:
Is `trackingCorrelationId` optional for the post endpoint? If it is not, then you should change this definition since the `trackingCorrelationId` is optional in `RaffleParticipation` type.

Reply:
Yes, it is.